.gitignore

```diff
@@ -1,4 +1,4 @@
-Donut/config.py
+donut/config.py
```

Comment:
Looks like the rename was reverted in this branch. If you just have `config.py` in the `.gitignore` then it'll ignore any file that is named `config.py`, regardless of which directory it is in.